### ISSUE TITLE
Mixer window is shifted out of the screen, no multi monitor

### ISSUE THREAD

**Issue by makedir** [File-New-Project/EarTrumpet#397]
Just turned on the laptop from hibernate and now the mixer window is shifted out of the screen. I dont have multiple monitors connected.

https://i.imgur.com/H1aRXGQ.png

**Comment by davuxcom** [comment]
Does restarting EarTrumpet fix the issue?  Do you see this each time you Hibernate the PC?

**Comment by davuxcom** [comment]
@makedir please let us know if you keep seeing this and can provide more information.

**Comment by lagraviereScience** [comment]
I confirm this "bug"/malfunction.

Screenshot of the problem:
![image](https://user-images.githubusercontent.com/9718556/206923558-3da18334-8f3d-4068-bb6c-f187ca5a4d57.png)
The image is from my monitor "0".

I have tried these things and it did not change the "bug"/malfunction:

- Tried having the same resolution on both screens (1920x1080).
  - My usual setup is monitor 0 is in 2560x1440 and monitor 1 is in 1920x1080.

- I tried switching the side of monitor 1 -> in Windows' settings I positioned monitor 1 on the left side.
  - My usual setup is with monitor 1 on the right.


If you want me to try other possibilities don't hesitate to ask me :)

**Comment by SubZero5268** [comment]
Just chiming with my experience as well! I found this thread while searching to see if anyone else encountered this issue. It happens to me, particularly while my display is set to my 4K TV. Resolution in Windows is set to 1080p 120hz. Restarting EarTrumpet fixes everything for a while, but eventually it'll happen again, usually after my PC goes to sleep or something like a game takes fullscreen focus.

Thankfully the only issue I've encountered with EarTrumpet, but still a bit of a pain.

![Eartrumpet Cutoff](https://user-images.githubusercontent.com/121411752/209491455-ce137a00-7c57-447d-ae21-0fa923797656.png)
